Services/DataBaseService/Utils/MigrationScripts/24.03.2020/1_CreateUsersTable.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE [Users] (
+    [Id]		UNIQUEIDENTIFIER NOT NULL,
+    [Email]		NVARCHAR (30)    NOT NULL,
+    [Password]  NVARCHAR (25)    NOT NULL,
```

Comment:
I think this stuff should be talked between frontend command. They set password with 500 symbols, as I know :)

Reply:
And also: Password Hash

Reply:
done